modules/packages/MPI.chpl

```diff
@@ -174,8 +174,13 @@ module MPI {
   use SysCTypes;
   require "mpi.h";
 
+
   use UtilReplicatedVar;
 
+  // TODO : This shouldn't need to be set by the user
+  // How do we query CHPL_COMM?
+  config param isUGNI=false;
```

Comment:
You should just be able to say `CHPL_COMM=="ugni"` 

CHPL_COMM is a builtin param string: (http://chapel.cray.com/docs/1.15/builtins/internal/ChapelEnv.html?highlight=chapelenv#ChapelEnv.CHPL_COMM)